src/routes/mod.rs

```diff
@@ -32,46 +32,51 @@ pub mod transactions;
 
 #[doc(hidden)]
 pub fn active_routes() -> Vec<Route> {
-    routes![
+    let no_openapi = routes![
         root,
+        contracts::routes::post_data_decoder,
```

Comment:
Why do the post roots don't have an openapi?

Reply:
I think  the question is more about why are `POST` routes excluded from Swagger. Or is really about the `/` endpoint @rmeissner ?

From what I could gather routes that specify a lifetime are not supported by `rocket_okapi`.

Reply:
Can we add a comment here linking to the issue about the lifetimes? I would have a group for `root` one for the endpoints which have lifetimes and another one for the ones that we do no want to expose

Reply:
You mean in code? Maybe something like this could be enough:
```
        root,
        // rocket_okapi don't support lifetimes
        // https://github.com/GREsau/okapi/issues/84
        contracts::routes::post_data_decoder,
        notifications::routes::post_notification_registration,
        safes::routes::post_safe_gas_estimation,
        safes::routes::post_safe_gas_estimation_v2,
        transactions::routes::post_confirmation,
        transactions::routes::post_transaction,
        // This endpoints shouldn't be exposed on swagger
        about::routes::redis,
        hooks::routes::update,
        hooks::routes::post_hook_update,
        hooks::routes::post_hooks_events,
        hooks::routes::post_flush_events,
        hooks::routes::flush
```

Reply:
I meant exactly this. No need to split in the codebase 👍